addons/remove-sprite-confirm/userscript.js

```diff
@@ -17,4 +17,7 @@ export default async ({ addon, console, msg }) => {
     );
     return Promise.resolve();
   };
+  vm.deleteSprite = newDeleteSprite;
+  addon.self.addEventListener("disabled", () => (vm.deleteSprite = oldDeleteSprite));
+  addon.self.addEventListener("reenabled", () => (vm.deleteSprite = newDeleteSprite));
```

Comment:
Same as above

Reply:
Here, it's not just about cleaner code - if 2 addons both pollute this, and because of luck this one polluted first, it means the second addon polluted this addon's pollution. If this addon does this, it might affect other addons, by removing their polluted method from the VM.

Reply:
These 2 lines should have been removed.